test/cpp/interop/interop_client.cc

```diff
@@ -83,7 +83,7 @@ CompressionType GetInteropCompressionTypeFromCompressionAlgorithm(
 }  // namespace
 
 InteropClient::InteropClient(std::shared_ptr<Channel> channel)
-    : channel_(channel) {}
+    : channel_(channel), stub_(TestService::NewStub(channel)) {}
```

Comment:
This is cleaner, but are we sure it doesn't change the semantics? Prior, the stub was re-created per testcase. This way, the same stub is reused.